doc/controls/DrawerFlyoutPresenter.md

```diff
@@ -15,26 +15,71 @@ xmlns:utu="using:Uno.Toolkit.UI"
        BasedOn="{StaticResource DrawerFlyoutPresenterStyle}"
        TargetType="FlyoutPresenter">
     <Setter Property="utu:DrawerFlyoutPresenter.OpenDirection" Value="Top" />
+    <Setter Property="utu:DrawerFlyoutPresenter.DrawerDepth" Value="0.66*" />
     <Setter Property="utu:DrawerFlyoutPresenter.LightDismissOverlayBackground" Value="#80808080" />
     <Setter Property="utu:DrawerFlyoutPresenter.IsGestureEnabled" Value="True" />
 </Style>
 <!-- and/or -->
 <utu:DrawerFlyoutPresenter OpenDirection="Top"
+                           DrawerDepth="0.66*"
                            LightDismissOverlayBackground="#80808080"
                            IsGestureEnabled="True" />
 ```
 
+> note:
+There is current a bug on windows that prevent the usage of attached property style setter. The workaround is to add the following code in your application:
+```xml
+<!--
+    microsoft/microsoft-ui-xaml#6388 (winui, and on windows only):
+    If you define attached property setter on a style with BasedOn another style with Template defined from a separate class library
+    it will throw when the template is materialized:
+    > Failed to assign to property 'Uno.Toolkit.UI.DrawerFlyoutPresenter.OpenDirection'.
+    ^ It will mention the first attached property used in the template, regardless of which attached property setter that triggered it.
+
+    The workaround here is to define the template in the consuming assembly again.
+    It doesn't matter if it is used or not.
+-->
+<win:Style x:Key="MUX6388_Workaround_ForDefinitionOnly" TargetType="FlyoutPresenter">
+    <Setter Property="Template">
+        <Setter.Value>
+            <ControlTemplate TargetType="FlyoutPresenter">
+                <!-- This is not the full template, we are just making explicit reference to these definitions below -->
+                <utu:DrawerFlyoutPresenter OpenDirection="{TemplateBinding utu:DrawerFlyoutPresenter.OpenDirection}"
+                                            DrawerDepth="{TemplateBinding utu:DrawerFlyoutPresenter.DrawerDepth}"
+                                            LightDismissOverlayBackground="{TemplateBinding utu:DrawerFlyoutPresenter.LightDismissOverlayBackground}"
+                                            IsGestureEnabled="{TemplateBinding utu:DrawerFlyoutPresenter.IsGestureEnabled}" />
+            </ControlTemplate>
+        </Setter.Value>
+    </Setter>
+</win:Style>
+```
+
 ### Properties
 Property|Type|Description
 -|-|-
 OpenDirection|DrawerOpenDirection|Gets or sets the direction in which the drawer opens toward.<br/>note: The position of drawer when opened is the opposite of this value.
+DrawerDepth|GridLength|Get or sets the depth (width or height depending on the `OpenDirection`) of the drawer.\*
```

Comment:
What's the asterisk for? I'm not seeing a footnote for it anywhere

Reply:
fell through the next section... :P